components/metrics/BUILD.gn

```diff
@@ -353,90 +376,91 @@ if (is_linux) {
     ]
   }
 }
+if (!use_cobalt_customizations) {
+  source_set("unit_tests") {
+    testonly = true
+    sources = [
+      "call_stack_profile_builder_unittest.cc",
+      "call_stack_profile_metrics_provider_unittest.cc",
+      "child_call_stack_profile_collector_unittest.cc",
+      "cloned_install_detector_unittest.cc",
+      "component_metrics_provider_unittest.cc",
+      "daily_event_unittest.cc",
+      "data_use_tracker_unittest.cc",
+      "drive_metrics_provider_unittest.cc",
+      "environment_recorder_unittest.cc",
+      "expired_histograms_checker_unittest.cc",
+      "field_trials_provider_unittest.cc",
+      "file_metrics_provider_unittest.cc",
+      "histogram_encoder_unittest.cc",
```

Comment:
Should select few of those be kept enabled ? histogram_encoder, metrics_service / manager etc look fairly relevant.
Of course it's fine to get the code in and do a separate follow-up for enabling the unit tests gradually - but i'd at least add a TODO bug here for a follow-up.

Reply:
+1 let's at least test if they run on linux before submitting this

Reply:
I've added a TODO to re-enable as much testing as possible.

Reply:
Is this something you're planning to work on relatively soon? I just want to check, as I often see this type of TODO never get addressed

Reply:
I guess that depends on your definition of "relatively soon". I will say, I'm quite keen to ensure telemetry does not break. As such, protecting the foundational code from inadvertent bugs/breakages is something I will do. When though... I'm not sure. The P0 here is getting Telemetry+UMA working. Then I'll go back and address bugs like this shortly after. If I had to provide an estimate, I'd estimate probably within 3 months at the latest... or thereabouts?

Reply:
Alright, just want to make sure it gets addressed eventually